curriculum/challenges/english/25-front-end-development/quiz-basic-html/66df3b712c41c499e9d31e5b.md

```diff
@@ -0,0 +1,976 @@
+---
+id: 66df3b712c41c499e9d31e5b
+title: Basic HTML Quiz
+challengeType: 8
+dashedName: basic-html-quiz
+---
+
+# --description--
+
+Answer all of the questions below correctly to pass the quiz.
+
+# --quizzes--
+
+## --quiz--
+
+### --quiz-question--
+
+How does the `loop` attribute work inside the `audio` element?
+
+### --question-options--
```

Comment:
maybe I missed this in the show component, but will the question options be randomized too?
because it looks like the answer will always be the first one

Reply:
The new structure changes this so the order doesn't matter. There's just "distractors" and "answer" - no numerical answer. The client will randomly order the distractors and answer.